src/vs/workbench/browser/parts/editor/editorPart.ts

```diff
@@ -404,7 +404,7 @@ export class EditorPart extends Part implements IEditorPart, IEditorGroupService
 			}
 
 			// Focus (unless prevented)
-			const focus = !options || !options.preserveFocus;
+			const focus = !options || options.preserveFocus;
```

Comment:
that is the correct behaviour. keeping focus should be the exception so please leave it like that